### ISSUE TITLE
[Docs] Problem of not being able to generate heat map in tool/test.py

### ISSUE THREAD

**Issue by SRAMXXX1** [open-mmlab/mmpose#2369]
### 📚 The doc issue

Thanks to the mmlab team for developing mmpose, it really works very well. I have a problem in using it.

It is able to get the Visualized heatmap easily in 'MMPose/demo/MMPose_Tutorial.ipynb'. But I can't get the heatmap by using 'tool/test.py'.  After investigation, I found that there is a difference in the model output of the two programs. The former output has the 'pred_fields' field while the latter does not, which is why the latter cannot produce a heatmap in 'local_visualizer.py add_datasample()'.

`        if draw_pred:
            pred_img_data = image.copy()
            pred_img_heatmap = None

            # draw bboxes & keypoints
            if 'pred_instances' in data_sample:
                pred_img_data = self._draw_instances_kpts(
                    pred_img_data, data_sample.pred_instances, kpt_thr,
                    show_kpt_idx, skeleton_style)
                if draw_bbox:
                    pred_img_data = self._draw_instances_bbox(
                        pred_img_data, data_sample.pred_instances)

            # draw heatmaps
            if 'pred_fields' in data_sample and draw_heatmap:
                if 'keypoint_x_labels' in data_sample.pred_instances:
                    pred_img_heatmap = self._draw_instance_xy_heatmap(
                        data_sample.pred_fields, image)
                else:
                    pred_img_heatmap = self._draw_instance_heatmap(
                        data_sample.pred_fields, image)
                if pred_img_heatmap is not None:
                    pred_img_data = np.concatenate(
                        (pred_img_data, pred_img_heatmap), axis=0)`

With more investigation, the model is built by init_pose_estimator() in 'demo/MMPose_Tutorial.ipynb' , while it is built by Runner.from_cfg(cfg) in 'tool/test.py'.  Is there any difference in the configuration method between the two that causes the 'pred_fields' field to be missing from the model output variables? 

In summary, I want to generate a visual heat map of the predictions for each sample in 'tool/test.py'. Eventually, I would like to see the heat map during the training of the network, just like I can see the results of each val through visulaizationhook. If this feature already exists, please let me know how to configure it.

**Comment by Ben-Louis** [comment]
To visualize heatmaps during testing, you could try adding `--cfg-options='model.test_cfg.output_heatmaps=True'` in your command

**Comment by SRAMXXX1** [comment]
> To visualize heatmaps during testing, you could try adding `--cfg-options='model.test_cfg.output_heatmaps=True'` in your command

Thank you for your answer. This method solves the problem. Also I found that I was actually able to get heat maps all the time.
It's just that my screen resolution is too small and the training image resolution is very large, and the window doesn't scale, so I never see the heat map below the color image.

Finally, thanks again for your answer!
